docs/pages/enroll-resources/database-access/enroll-self-hosted-databases/oracle-exadata.mdx

```diff
@@ -0,0 +1,428 @@
+
+```code
+# Connect to the VM
+$ ssh opc@demodb-vm
+# Switch to grid user
+$ sudo su - grid
+# Export the database certificate. Update the "-dn" parameter to match your actual setup.
+$ orapki wallet export -wallet "/var/opt/oracle/dbaas_acfs/grid/tcps_wallets" -dn "CN=demodb-vm-scan.exadatadomain.oke.oraclevcn.com"  -cert /tmp/oracle-server-certificate.crt
+```
+
+Save the `/tmp/oracle-server-certificate.crt` file to a suitable location. It will be used by the Database service when connectiong to the Oracle database. 
+The final location for this file depends on the Teleport installation method and will be explained in the next step.
+
+As the `grid` user, check the TCPS address of local listener. This will be the address that Teleport will use for connecting. In the example below, that address is <Var name="10.20.30.40:2484" />.
+
+```code
+$ sqlplus / as sysdba
+# ...
+# SQL> SHOW PARAMETER local_listener;
+# 
+# . NAME           TYPE   VALUE
+# . -------------- ------ ------------------------------
+# . local_listener string (ADDRESS=(PROTOCOL=TCP)(HOST=10.20.30.40)(PORT=1521)),
+# .                       (ADDRESS=(PROTOCOL=TCPS)(HOST=10.20.30.40)(PORT=2484))
+```
+
+## Step 4/6. Configure and Start the Database Service
+Install and configure Teleport where you will run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Server">
+
+(!docs/pages/includes/install-linux-enterprise.mdx!)
+
+On the host where you will run the Teleport Database Service, start Teleport
+with the appropriate configuration.
+
+Note that a single Teleport process can run multiple different services, for
+example multiple Database Service agents as well as the SSH Service or Application
+Service. The step below will overwrite an existing configuration file, so if
+you're running multiple services add `--output=stdout` to print the config in
+your terminal, and manually adjust `/etc/teleport.yaml`.
+
+Copy the Oracle Database certificate and make it available at
+   `/var/lib/teleport/oracle-server-certificate.crt` on the Teleport Database Service host.
+
+Run the following command to generate a configuration file at
+`/etc/teleport.yaml` for the Database Service. Update 
+<Var name="example.teleport.sh" /> to use the host and port of the Teleport Proxy
+Service:
+```code
+$ sudo teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=<Var name="example.teleport.sh:443" /> \
+   --name="<Var name="oracle" />" \
+   --protocol=oracle \
+   --uri="<Var name="10.20.30.40:2484" />" \
+   --ca-cert-file="var/lib/teleport/oracle-server-certificate.crt" \
+   --labels=env=dev
+```
+
+The final entry for the `<Var name="oracle" />` database will look similar to this:
+
+```yaml
+db_service:
+  enabled: true
+  databases:
+  - name: <Var name="oracle" />
+    uri: "<Var name="10.20.30.40:2484" />"
+    protocol: oracle
+    tls:
+      mode: verify-ca
+      ca_cert_file: /var/lib/teleport/oracle-server-certificate.crt
```

Comment:
So this is CA certificate - then per my comments above I would rephrase in a couple of places to make this more explicit.